rust/ReadMe.md

```diff
@@ -1,9 +1,10 @@
 # Smartbots 2023 Starter Code in Rust
 
-This is a basic bot made with  [js-common](https://github.com/Bhoos/smartbots-2023-starter-code/tree/master/js-common) taken as a reference. This should be taken as a boilerplate on top of which you should implement your logic. 
-
+This is an improved starter-code in Rust, everything is made again from scratch taking reference of old version.
```

Comment:
@krischal111 Refrain from using words like "improved", since "improvement" for a starter code is highly subjective. Use words like "updated" instead.

Reply:
Okay